pkg/apis/operator/v1/commonaudit_types.go

```diff
@@ -0,0 +1,114 @@
+//
+// Copyright 2020 IBM Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+package v1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// EDIT THIS FILE!  THIS IS SCAFFOLDING FOR YOU TO OWN!
+// NOTE: json tags are required.  Any new fields you add must have json tags for the fields to be serialized.
+
+// CommonAuditSpec defines the desired state of CommonAudit
+type CommonAuditSpec struct {
+	// EnableAuditLoggingForwarding defines if audit logs should be forwarded to an SIEM or not
+	EnableAuditLoggingForwarding bool                   `json:"enabled,omitempty"`
+	ClusterIssuer                string                 `json:"clusterIssuer,omitempty"`
+	Fluentd                      CommonAuditSpecFluentd `json:"fluentd,omitempty"`
+	Outputs                      CommonAuditSpecOutputs `json:"outputs,omitempty"`
+}
+
+// CommonAuditSpecFluentd defines the desired state of Fluentd
+type CommonAuditSpecFluentd struct {
+	ImageRegistry string                   `json:"imageRegistry,omitempty"`
+	PullPolicy    string                   `json:"pullPolicy,omitempty"`
+	Replicas      int                      `json:"replicas,omitempty"`
```

Comment:
@lauriet-ibm im not sure where we are supposed to put the "replicas" field...according to this doc it should be in the outermost spec https://github.ibm.com/IBMPrivateCloud/BedrockServices/blob/master/OverallRequiredCapabilities/OperatorChangesForSizing.md#crd-spec-single-workload